cadence/contracts/FlowYieldVaultsStrategies.cdc

```diff
@@ -617,11 +618,11 @@ access(all) contract FlowYieldVaultsStrategies {
     fun _createRecurringConfig(withID: DeFiActions.UniqueIdentifier?): DeFiActions.AutoBalancerRecurringConfig {
         // Create txnFunder that can provide/accept FLOW for scheduling fees
         let txnFunder = self._createTxnFunder(withID: withID)
-        
+
         return DeFiActions.AutoBalancerRecurringConfig(
-            interval: 60,  // Rebalance every 60 seconds
+            interval: 60 * 60 * 30,  // Rebalance every 30 minutes
             priority: FlowTransactionScheduler.Priority.Medium,
-            executionEffort: 800,
+            executionEffort: 999,
```

Comment:
why is this updated to 999? the new logic added from this PR increased the execution effort?